packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+import { authenticationProviders } from '../../stores/authenticationProviders';
+import { faCircle, faRightFromBracket } from '@fortawesome/free-solid-svg-icons';
+import SettingsPage from './SettingsPage.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import KeyIcon from '../images/KeyIcon.svelte';
+</script>
+
+<SettingsPage title="Authentication">
+  <div class="containder h-full">
+    <!-- Authentication Providers table start -->
+    <div class="w-full h-full">
+      {#if $authenticationProviders.length === 0}
+        <div class="grid grid-cols-1 justify-items-center content-center h-full w-full">
+          <div class="mb-6"><svelte:component this="{KeyIcon}" size="55" /></div>
+          <div><h1 class="pf-c-title pf-m-lg text-center">No authentication providers</h1></div>
+          <div class="text-gray-400 mx-16 mt-3 text-center w-q">
+            Install an authentication provider extension to add an authentication provider here.
+          </div>
```

Comment:
you need to use <EmptyScreen> component

like for example in ContainerDetailsLogs page

```

<EmptyScreen
  icon="{NoLogIcon}"
  title="No Log"
  message="Log output of {container.name}"
  hidden="{noLogs === false}"
  style="background-color: {getPanelDetailColor()}" />
```

Reply:
I could not figure out why the icon was not aligned correctly, when using EmptyScreen component, that is why I implemented it this way. I figured it out this time.